extensions/schema-compare/src/dialogs/schemaCompareOptionsDialog.ts

```diff
@@ -264,17 +264,17 @@ export class SchemaCompareOptionsDialog {
 
 
 			this.optionsTable = view.modelBuilder.table().component();
-			this.updateOptionsTable();
+			await this.updateOptionsTable();
 
 			this.disposableListeners.push(this.optionsTable.onRowSelected(async () => {
 				let row = this.optionsTable.selectedRows[0];
 				let label = this.optionsLabels[row];
-				this.descriptionText.updateProperties({
+				await this.descriptionText.updateProperties({
 					value: this.GetDescription(label)
 				});
 			}));
 
-			this.disposableListeners.push(this.optionsTable.onCellAction(async (rowState) => {
+			this.disposableListeners.push(this.optionsTable.onCellAction((rowState) => {
```

Comment:
Why was "async" removed from here?

Reply:
nothing is being awaited in the callback

Reply:
How about line 295, where an "await" is added?